lib/explorer/data_frame.ex

```diff
@@ -1744,6 +1769,30 @@ defmodule Explorer.DataFrame do
         a integer [3, 3, 1]
         b integer [2, 3, 1]
       >
+
+  ## Grouped examples
+
+  When used in a grouped dataframe, `arrange_with/2` is going to sort each group individually
+  and then return the entire dataframe with the existing groups. If one of the arrange columns
+  is also a group, the sorting for that column is not going to work. It is necessary to
```

Comment:
What do you mean by "it is not going to work?". Does it raise? Does it return the wrong value?